pyscriptjs/src/components/base.ts

```diff
@@ -41,22 +42,22 @@ export class BaseEvalElement extends HTMLElement {
         this.shadow = this.attachShadow({ mode: 'open' });
         this.wrapper = document.createElement('slot');
         this.shadow.appendChild(this.wrapper);
-        this.setOutputMode("append");
+        this.setOutputMode('append');
     }
 
     addToOutput(s: string) {
         this.outputElement.innerHTML += '<div>' + s + '</div>';
         this.outputElement.hidden = false;
     }
 
-    setOutputMode(defaultMode = "append") {
+    setOutputMode(defaultMode = 'append') {
         const mode = this.hasAttribute('output-mode') ? this.getAttribute('output-mode') : defaultMode;
 
         switch (mode) {
-            case "append":
+            case 'append':
                 this.appendOutput = true;
                 break;
-            case "replace":
+            case 'replace':
```

Comment:
Unrelated to this PR but maybe be it could be useful to add prettier or some other JS formatted to pre-commit to avoid such formatting changes?

Reply:
Great call. Opened #505

Reply:
Ah agreed. I had seen the `format` script in the projects package.json file (`"prettier --write ./src/**/*.{js,svelte,html,ts}'"`) and assumed I should run it before committing. Apologies for the mess.